lua/entities/dakara_building.lua

```diff
@@ -299,18 +299,26 @@ function ENT:OnRemove(ply)
 	if IsValid(self.SecretButton2) then self.SecretButton2:Remove() end
 end
 
-function ENT:Think(ply)
-	if (not IsValid(self)) then return end
-	local e = ents.FindInSphere(self:GetPos()+Vector(0,0,1000), 1000);
-	for _,v in pairs(e) do
-		if (v:IsPlayer() and v:GetMoveType() == MOVETYPE_NOCLIP) then
-			v:SetMoveType(MOVETYPE_WALK)
+local PlayersInDakara = {} -- players inside a dakara building
```

Comment:
There can be even more problems - if you remove dakara all players what was inside will not restore their noclip.